### DISCUSSION TITLE
Additional potentiometer / slider for data entry?

### DISCUSSION THREAD

**Discussion by fp64lib** [probonopd/MiniDexed#164] [Ideas]
I like the approach of MiniDexed to come with a minmal hardware setup: a Raspi, an audio card, a display and one rotary encoder. However, at least in my usage, you soon come to the point where you are doing a lot of turning and turning for parameter adjustments. Have you ever tried to turn down the volume on 4 voices in order to hear and tweak the remaining voice#5?

When I look at other synths, you have also a potentiometer (rotating or sliding) to enter/modify values, which speeds up the workflow especially when you are making bigger changes. On the DX7, this slider was labelled "data entry". So, this would also fit very good to the concept of having a mini DX7.

What do you think? Should there be a slider/potentiometer for modifying data in the standard setup of the MiniDexed? There are still some GPIOs left, so it can work from regarding hardware ressources.

**Comment by craigyjp** [comment]
How far do you want to take this? I don't see the point, ok the DX7 had the
32 buttons and then data entry slider to adjust things, but why does having
two encoders speed up editing really?

Probably takes longer to move your hand between encoders than it does to
click on the parameter selected.

Keep it minimal and let's hope we get editing over midi one day to really
open up the synth.

Remember although it's a DX7 it works more like a TX816 which has pretty
much zero controls.

**Reply by JohnConnor84** [reply]
I was looking a 4x4 membrane keypad...it connects direct to the GPIO ports and it looks very 80s. Do you guys think it can be configured as 4 navigation arrows, back button and the rest assigned to other basic functions?  My idea would be the membrane keypad at the left side of the LCD and the rotary encoder at the right, so you could easily navigate through.

**Comment by probonopd** [comment]
I think using the knobs and sliders on MIDI controllers is the way to go. Maybe we can even get rid of the one remaining rotary encoder. Reduce to the max!
